.github/workflows/runtime-validation-tests-pipeline.yml

```diff
@@ -0,0 +1,20 @@
+
+name: Runtime Validation Tests Pipeline
+
+on: [push]
```

Comment:
do we want to run this on **every** push or jush PRs: https://docs.github.com/en/actions/writing-workflows/choosing-when-your-workflow-runs/events-that-trigger-workflows#pull_request